### ISSUE TITLE
New emojis are not supported

### ISSUE THREAD

**Issue by sujay-bidchat** [omnidan/node-emoji#69]
@omnidan Please add support to New emojis in Android & iOS.

**Comment by kabdulahi** [comment]
https://www.npmjs.com/package/node-emoji#adding-new-emoji

When following these instruction you should get the new emojies that were not in this package yet. It worked for me!

**Comment by tracycollins** [comment]
Maybe I'm just dense, or the caffeine hasn't kicked in yet, but I can figure out how to follow the instructions to add emoji.

Specifically, the "rolling_on_the_floor_laughing": "🤣" emoji isn't being recognized, and I'm trying to update the list.

The instructions say:

> You can get a JSON file with all emoji here: https://raw.githubusercontent.com/omnidan/node-emoji/master/lib/emoji.json

The emoji.json file is of the form:

`{
 "rolling_on_the_floor_laughing": "🤣"
}`

**Comment by kabdulahi** [comment]
**Solution: ** Overwrite emoji.json with https://raw.githubusercontent.com/omnidan/node-emoji/master/lib/emoji.json and do _not_ parse. 

So I think the instructions that are on the package's page are indeed very confusing. So I tried again to fix this problem, but I did not succeed. So why did it work the first time for me? I don't know but I probably did the following by accident. I checked the emojiparse.js file and apparently it reads emojifile.js parses it and outputs to emoji.json. Then I suppose we can just replace the emoji.json file from the file omnidan repo and do nothing? So that's what I did and it worked :)
